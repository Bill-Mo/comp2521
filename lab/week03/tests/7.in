10 5 12 3 6 7 9
