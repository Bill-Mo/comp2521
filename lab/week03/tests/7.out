Tree:
   10
   / \
  5  12
 / \
3   6
     \
      7
       \
        9

#nodes = 7,  #leaves = 3
Infix  : 3 5 6 7 9 10 12 
Prefix : 10 5 3 6 7 9 12 
Postfix: 3 9 7 6 5 12 10 
ByLevel: 10 5 12 3 6 7 9 
